pkg/cmd/server/api/types.go

```diff
@@ -83,6 +95,8 @@ const (
 	ControllersAll = "*"
 )
 
+type FeatureSet map[OpenShiftFeature]util.Empty
```

Comment:
As far as naming - based on discussion today it would stay "oadm".  Does
the pull stay that way?

On Mon, Aug 3, 2015 at 10:47 AM, Michal Minar <notifications@github.com>
wrote:

> In pkg/cmd/server/api/types.go
> <https://github.com/openshift/origin/pull/3953#discussion_r36091997>:
>
> > @@ -83,6 +95,8 @@ const (
> >  	ControllersAll = "*"
> >  )
> >
> > +type FeatureSet map[OpenShiftFeature]util.Empty
>
> I see, thanks for an explanation
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3953/files#r36091997>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift